hammer/README.md

```diff
@@ -0,0 +1,27 @@
+# Hammer: A load testing tool for Tessera logs
+
+This hammer sets up read and (optionally) write traffic to a log to test correctness and performance under load.
+The read traffic is sent according to the [tlog-tiles](https://github.com/C2SP/C2SP/blob/main/tlog-tiles.md) spec, and thus could be used to load test any tiles-based log, not just Tessera logs.
+
+If write traffic is enabled, then the target log must support `POST` requests to a `/add` path.
```

Comment:
Should we add that the returned body must be an ASCII decimal number representing the assigned index?